wit/wasi-filesystem.wit.md

```diff
@@ -228,84 +228,84 @@ record directory-entry {
 
 ## `error-code`
 ```wit
-/// Error codes returned by functions.
+/// Error codes returned by functions, similar to `errno` in POSIX.
 /// Not all of these error codes are returned by the functions provided by this
 /// API; some are used in higher-level library layers, and others are provided
 /// merely for alignment with POSIX.
 enum error-code {
-    /// Permission denied.
+    /// Permission denied. Similar to `EACCES` in POSIX.
     access,
-    /// Resource unavailable, or operation would block.
+    /// Resource unavailable, or operation would block. Similar to `EAGAIN` and `EWOULDBLOCK` in POSIX.
     would-block,
-    /// Connection already in progress.
+    /// Connection already in progress. Similar to `EALREADY` in POSIX.
     already,
-    /// Bad descriptor.
+    /// Bad descriptor. Similar to `EBADF` in POSIX.
     bad-descriptor,
-    /// Device or resource busy.
+    /// Device or resource busy. Similar to `EBUSY` in POSIX.
     busy,
-    /// Resource deadlock would occur.
+    /// Resource deadlock would occur. Similar to `EDEADLK` in POSIX.
     deadlock,
-    /// Storage quota exceeded.
+    /// Storage quota exceeded. Similar to `EDQUOT` in POSIX.
     quota,
-    /// File exists.
+    /// File exists. Similar to `EEXIST` in POSIX.
     exist,
-    /// File too large.
+    /// File too large. Similar to `EFBIG` in POSIX.
     file-too-large,
-    /// Illegal byte sequence.
+    /// Illegal byte sequence. Similar to `EILSEQ` in POSIX.
     Illegal-byte-sequence,
```

Comment:
Why not
```suggestion
    illegal-byte-sequence,
```
Two things bug me:
1. the upper `I` here stands out from the lower starts of all the other names 
2. it is also quite close to what the following lower `l` looks like

Reply:
Good catch, fixed in 84e2a7f65c40ac14a7d23771dda536252d1d6176, thanks for reporting this typo.